SPECS/python-libclang/python-libclang.spec

```diff
@@ -0,0 +1,41 @@
+%global pypi_name libclang
+
+Summary:        Clang's python bindings
+Name:           python-%{pypi_name}
+Version:        14.0.6
+Release:        1%{?dist}
+License:        ASL 2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/sighingnow/libclang
+Source0:        %{pypi_source}
```

Comment:
%pypi_source is not defined

Reply:
Neha %{pypi_source} macro can be used in source for python project in place of an actual source URL. 
We need pypi_name variable declared somewhere. It will also use version variable to infer the source URL.